daemon/start.go

```diff
@@ -176,19 +176,35 @@ func (daemon *Daemon) containerStart(container *container.Container, checkpoint
 		return err
 	}
 
-	err = daemon.containerd.Create(context.Background(), container.ID, spec, createOptions)
+	ctx := context.TODO()
```

Comment:
For my understanding: Why TODO context vs background?

Reply:
Because we should figure out a way to plumb a proper context through here, otherwise the context is useless and is just being passed in to satisfy the argument list.

Reply:
I've seen a lot of code where `context.Background` is passed simply to satisfy the argument. Glad to see TODO being used.